### ISSUE TITLE
Plugin incompatible with kotlin version 2.0.20

### ISSUE THREAD

**Issue by AlbRoehm** [tabilzad/ktor-docs-plugin#26]
When using the kotlin v2.0.20 (via gradle plugin "org.jetbrains.kotlin.jvm" version 2.0.20) the `compileKotlin` build step is interrupted by the following exception. Going back to v2.0.10 or v.2.0.0 resolves this issue. 

```
java.lang.NoClassDefFoundError: org/jetbrains/kotlin/ir/util/IrMessageLoggerKt
	at io.github.tabilzad.ktor.k2.SwaggerDeclarationChecker.<init>(SwaggerDeclarationChecker.kt:29)
	at io.github.tabilzad.ktor.FirCheckers$declarationCheckers$1.<init>(KtorMetaPluginRegistrar.kt:40)
	at io.github.tabilzad.ktor.FirCheckers.<init>(KtorMetaPluginRegistrar.kt:37)
	at io.github.tabilzad.ktor.FirCheckers$Companion.getFactory$lambda$0(KtorMetaPluginRegistrar.kt:45)
```

**Comment by tabilzad** [comment]
Thanks for reporting, will take a look.

**Comment by tabilzad** [comment]
0.6.4 has been published with support for 2.0.20. It should be available in maven tomorrow.
